### DISCUSSION TITLE
Digitalocean deployment ERR_CONNECTION_REFUSED

### DISCUSSION THREAD

**Discussion by ivoba** [mozilla/hubs#2817] [Hubs Cloud Questions] [answered]
Hi, i followed exactly the instructions on https://hubs.mozilla.com/docs/hubs-cloud-do-quick-start.html

The setup script says: All set! Your hub will be available in a few minutes at https://hubs.mydomain.net.

However when opening the url the browser says: ERR_CONNECTION_REFUSED

Ping also has 100% packet loss, but IP is the correct one from Digitalocean.

What could have gone wrong?

**Comment by robinkwilson** [comment, marked as answer]
Is this still unsolved? Or did you figure it out? 

Will set as "answered" please mention me in another comment if this is unresolved.

**Reply by ivoba** [reply]
@robin-k-wilson we couldnt solve it :( and are now using the public hubs service.
I'm sorry we did not have the resources to debug this in depth.

**Comment by NickNaglich** [comment]
I just setup a new digitalocean dropplet and am having this same issue where it fails to connect after successfully running the setup script. Before the script the webpage was coming up saying to follow the setup process.
I figured out how to ping it by adding ICMP to the firewall settings, but it still won't connect to the webpage.

Any help would be greatly appreciated, because I don't see a whole lot of documentation about setting up a digitalocean server.

**Comment by chandeepadissanayake** [comment]
This issue still persists and any help would be appreciated.
